src/HTTP.jl

```diff
@@ -301,42 +322,46 @@ After a suspected connection task failure:
     end
 ```
 """
-function serve(server::ServerWS{T, H, W}, host, port, verbose) where {T, H, W}
-
+function serve(server::ServerWS{T, H, W}, host, port, verbose, sleeptime = 0.01) where {T, H, W}
```

Comment:
Good idea to include sleeptime. We used to completely avoid keyword arguments in earlier Julia versions, but if somebody finds the time to incorporate it, there are only upsides now.